pkg/pgmodel/sql_ingest.go

```diff
@@ -462,6 +465,8 @@ func (h *insertHandler) blockingHandleReq() bool {
 			return true
 		case <-h.seriesCacheRefresh.C:
 			h.refreshSeriesCache()
+		case <-h.epochAbort:
```

Comment:
I might put this first in the SELECT since we want to do this before processing any more input if it's non-empty